Add Adyen to dashboard docs (#165)
mirumee/saleor-docs: docs/dashboard/configuration/plugins.md

  - Stripe

>   - Adyen

Reviewer: Could we maybe sort them?

Author: Yes! 👮

> - Used to extend the Saleor functionalities

Reviewer: ```suggestion
- Used to provide certain core features:
```

  
  - Webhooks

  - Invoicing

In all cases of plugin configuration, you need to provide your credentials. Depending on the gate you are configuring, there might be additional setup required.

## Adyen

### Enabling the adyen integration

> - Go to Configuration -> Plugins -> Adyen -> Fill up the fields

Reviewer: ```suggestion
Go to _Configuration_ -> _Plugins_ -> _Adyen_ and fill in the fields:
```

> - `Merchant Account` - your Adyen account as described [here](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)

Reviewer: ```suggestion
- **Merchant Account:** your Adyen account identifier as described in the [getting started with Adyen guide](https://docs.adyen.com/checkout/get-started#step-1-sign-up-for-a-test-account)
```

> - `Supported currencies` - your currency in ISO 4217 like `GBP`

Reviewer: ```suggestion
- **Supported currencies:** your currency as an ISO 4217 3-letter code (eg. `USD`, `EUR`, `GBP`)
```

> - `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs.](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key)

Reviewer: ```suggestion
- `Client Key` - The client key is a public key that uniquely identifies a web service. [See Adyen docs](https://docs.adyen.com/user-management/client-side-authentication#get-your-client-key).
```

> - `Origin Url` - The origin URL of the page where you are rendering the Drop-in. The origin URL of the page where you are rendering the Drop-in. This should not include subdirectories and a trailing slash. For example, if you are rendering the Drop-in on `https://your-company.com/checkout/payment`, specify here: `https://your-company.com`. [See Adyen docs.](https://docs.adyen.com/checkout/drop-in-web)"

Reviewer: ```suggestion
- **Origin URL:** The origin URL of the page where you are rendering the Adyen Drop-in. This should not include subdirectories or a trailing slash. For example, if you are rendering the Drop-in on `https://your-company.com/checkout/payment`, put in `https://your-company.com` as the origin. See [Adyen docs](https://docs.adyen.com/checkout/drop-in-web).
```

> - `Live` - Leave it blank when you want to use test env. [See Adyen docs for production env](https://docs.adyen.com/development-resources/live-endpoints)

Reviewer: ```suggestion
- **Live:** Leave it blank if you want to use the test environment. See [Adyen docs for production environments](https://docs.adyen.com/development-resources/live-endpoints).
```

> - `Assume all authorizations are automatically captured by Adyen` - All authorized payments will be marked as captured. This should only be enabled if Adyen is configured to auto-capture payments. Saleor doesn’t support the delayed capture Adyen feature.

Reviewer: ```suggestion
- **Assume all authorizations are automatically captured by Adyen:** All authorized payments will be marked as captured. This should only be enabled if Adyen is configured to auto-capture payments. Saleor doesn’t support the _delayed capture_ Adyen feature.
```

> - `Automatically capture funds when a payment is made` - If enabled, Saleor will automatically capture funds. If disabled, the funds are blocked but need to be captured manually.

Reviewer: ```suggestion
- **Automatically capture funds when a payment is made:** If enabled, Saleor will automatically capture funds. If disabled, the funds are blocked but need to be captured manually.
```

> - `HMAC secret key` - key used to check if a received notification comes from Adyen. The key should be the same as on the Adyen side. Generate new key on the customer-area and copy it to Saleor. [See Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)

Reviewer: ```suggestion
- **HMAC secret key:** key used to check if a received notification comes from Adyen. The key should be the same as on the Adyen side. Generate a new key in the Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

> - `Notification user` - Additionally you can confirm received notification by checking the provided user in notification. Provide user in Authentication section in your customer area and copy it to Saleor. [See Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)

Reviewer: ```suggestion
- **Notification user:** In addition to checking request signatures, you can confirm received notification by checking the provided user in the notification. Provide a user in the _Authentication_ section of your Adyen customer area and copy it here. See [Adyen docs](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

> - `Notification password` - Additionally you can confirm received notification by checking the provided user’s password in notification . Provide user’s password in Authentication section in your customer area and copy it to Saleor. [`See Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area)

Reviewer: ```suggestion
- **Notification password:** In addition to checking request signatures, you can confirm received notification by checking the provided user’s password in the notification . Provide a password in the _Authentication_ section of your Adyen customer area and copy it here. See [`Adyen docs`](https://docs.adyen.com/development-resources/webhooks#set-up-notifications-in-your-customer-area).
```

> Below you can find the sample configuration of the Adyen.

Reviewer: ```suggestion
Below you can find an example configuration of the Adyen plugin:
```
